remote_docs_config.json

```diff
@@ -1,7 +1,26 @@
 {
-  "Piwik_PRO_Library_for_Vue.md": {"url": "https://raw.githubusercontent.com/PiwikPRO/vue-piwik-pro/master/README.md", "path": "data_collection/web/frameworks/"},
-  "Piwik_PRO_Library_for_Angular.md": {"url": "https://raw.githubusercontent.com/PiwikPRO/ngx-piwik-pro/master/README.md", "path": "data_collection/web/frameworks/"},
-  "Piwik_PRO_Library_for_React.md": {"url": "https://raw.githubusercontent.com/PiwikPRO/react-piwik-pro/master/README.md", "path": "data_collection/web/frameworks/"},
-  "Piwik_PRO_Library_for_NextJS.md": {"url": "https://raw.githubusercontent.com/PiwikPRO/next-piwik-pro/master/README.md", "path": "data_collection/web/frameworks/"},
-  "Piwik_PRO_Plugin_for_Gatsby.md": {"url": "https://raw.githubusercontent.com/PiwikPRO/gatsby-plugin-piwik-pro/master/README.md", "path": "data_collection/web/frameworks/"}
-}
\ No newline at end of file
+  "Piwik_PRO_Library_for_Vue.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/vue-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  },
+  "Piwik_PRO_Library_for_Angular.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/ngx-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  },
+  "Piwik_PRO_Library_for_React.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/react-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  },
+  "Piwik_PRO_Library_for_NextJS.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/next-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  },
+  "Piwik_PRO_Plugin_for_Gatsby.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/gatsby-plugin-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  },
+  "Piwik_PRO_Library_for_NuxtJS.md": {
+    "url": "https://raw.githubusercontent.com/PiwikPRO/nuxt-piwik-pro/master/README.md",
+    "path": "data_collection/web/frameworks/"
+  }
```

Comment:
I ran `python3 download_remote_docs.py` and added that file so this pr will have also initial document.